.github/workflows/lint.yml

```diff
@@ -0,0 +1,24 @@
+name: Linting
```

Comment:
Please remove this -we don't use GitHub Actions for linting as it's built into our CircleCI pipeline.